reference/docs-conceptual/SecretManagement/how-to/manage-secretstore.md

```diff
@@ -0,0 +1,157 @@
+---
+description: Explains how to manage a local SecretStore vault
+ms.date: 05/24/2022
+title: Managing a SecretStore vault
+---
+# Managing a SecretStore vault
+
+The **SecretStore** module is an extension vault for the PowerShell **SecretManagement** module. It
+stores secrets, locally, in files for the current user account context, and uses .NET crypto APIs to
+encrypt file contents. The **SecretStore** module has several configuration options. In the default
+configuration, a password is required to store and access secrets, and provides the strongest
+protection. **SecretStore** also supports the storage of metadata about secrets.
+
+## Registering a new vault
+
+Before you can create new secret you must register a vault. The **Name** parameter is a friendly
+name and can be anything you choose.
```

Comment:
Nit: This implies the value of **Name** _could_ be a hashtable or some other object.
```suggestion
name and can be any valid string.
```